Build/PreBuild.ps1

```diff
@@ -8,7 +8,8 @@ $need_symlink = @(
     "python2",
     "MicroBuild.Core",
     "Microsoft.VSSDK.BuildTools",
-    "Newtonsoft.Json"
+    "Newtonsoft.Json",
+    "Microsoft.VisualStudio.Python.LanguageServer"
```

Comment:
Is this necessary?

Reply:
Oh I see it's so that this doesn't require the version in the path: `<TypeshedFiles Include="$(PackagesPath)\Microsoft.VisualStudio.Python.LanguageServer\lib\vs16\Typeshed\**\*" />`